onboarding_tutorial.py

```diff
@@ -0,0 +1,77 @@
+class OnboardingTutorial:
```

Comment:
Please add a comment at the top of this file linking to the tutorial you got it from

Reply:
The comment has been added.

#slackbot tutorial original code from https://github.com/slackapi/python-slack-sdk/tree/main/tutorial, modified.